etc/spark-jobserver.conf

```diff
@@ -0,0 +1,18 @@
+spark {
+  master = "local[4]"
+
+  job-number-cpus = 4
```

Comment:
Should the CPU count be templated? boot2docker made me an 8 cpu VM to match my core count.